src/CMakeLists.txt

```diff
@@ -511,7 +511,30 @@ set_target_properties(${target}
 
 list(TRANSFORM HEADERS PREPEND "${CMAKE_CURRENT_LIST_DIR}/")
 set_target_properties(${target} PROPERTIES PUBLIC_HEADER "${HEADERS}")
+set_target_properties(${target} PROPERTIES EXPORT_NAME ${PROJECT_NAME})
```

Comment:
@uwerat @lgtmak this introduces an additional complexity where internal cmake targets are named differently from external/consumed. I would rather name the internal target the same way a user would address/consume them...

e.g.

internal target name for the "core library" `qskinny` vs. `TBD::QSkinny` for a consumer where `TBD` stands for the QSkinny framework's package namespace discussed elsewhere

Reply:
Addressed in #358